server/pages/index.jade

```diff
@@ -44,16 +44,34 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 		link(rel='apple-touch-icon', sizes='180x180', href='//s1.wp.com/i/favicons/apple-touch-icon-180x180.png')
 		link(rel='profile', href='http://gmpg.org/xfn/11')
 		link(rel='manifest', href='/calypso/manifest.json')
-		link(rel='stylesheet', href='https://fonts.googleapis.com/css?family=Noto+Serif:400,400i,700,700i&subset=cyrillic,cyrillic-ext,greek,greek-ext,latin-ext,vietnamese')
-		link(rel='stylesheet', href='//s1.wp.com/i/noticons/noticons.css?v=20150727')
-		link(rel='stylesheet', href='//s1.wp.com/wp-includes/css/dashicons.css?v=20150727')
+
+		if !! renderedLayout
+			//- If we're server-side rendering, just load the stylesheets normally
+			- cssLinkAttrs = { rel: 'stylesheet' }
+		else
+			//- If we're not server-side rendering, load CSS async
+			- cssLinkAttrs = { rel: 'preload', as: 'style', onload: "this.rel='stylesheet'" }
+
+		link(href='https://fonts.googleapis.com/css?family=Noto+Serif:400,400i,700,700i&subset=cyrillic,cyrillic-ext,greek,greek-ext,latin-ext,vietnamese')&attributes( cssLinkAttrs )
+		link(href='//s1.wp.com/i/noticons/noticons.css?v=20150727')&attributes( cssLinkAttrs )
+		link(href='//s1.wp.com/wp-includes/css/dashicons.css?v=20150727')&attributes( cssLinkAttrs )
+
+		if ! renderedLayout
```

Comment:
@DanReyLop Alright, I looked into this and rediscovered the problem I was facing.

When I put the `shell.css` _above_ the `noticons.css` link, the W logo is not the right size. This is because the `font-size` declaration in the two CSS files conflict (both match a single class, so they have the same specificity). We need to use `shell.css` for it to be the right size, and the simplest way was to include `shell.css` _later_ in the HTML than `noticons.css`.

I moved the polyfill declaration though. I agree that it makes sense for that to be with the `cssLinkAttrs` definition. But if we also want to move `shell.css` up there, we're probably going to have to play with CSS classes to make sure `shell.css`'s rule for the logo is more specific than the rule in `noticons.css`.